wiki/Beatmap/Difficulty/id.md

```diff
@@ -5,25 +5,24 @@ tags:
   - SR
   - star rating
   - stars
-outdated: true
 ---
 
 # Tingkat kesulitan
 
-*Artikel ini tidak membahas seputar [Beatmap](/wiki/Beatmap) secara umum.*
+*Berbeda konteks dengan [Beatmap](/wiki/Beatmap).*
 
-Istilah **tingkat kesulitan beatmap** (*beatmap difficulty*) secara garis besar merujuk pada berkas-berkas [dengan ekstensi `.osu`](/wiki/osu!_File_Formats/Osu_(file_format)) yang tersimpan di dalam folder masing-masing beatmap. Berkas-berkas ini merupakan bentuk manifestasi dari level-level permainan yang ada dan berisikan antara lain informasi seputar [hit object](/wiki/Hit_object) pada [mode permainan](/wiki/Game_mode) yang terkait serta [timing](/wiki/Beatmapping/Timing_section) dari lagu yang dipergunakan. Pada umumnya, seberapa sukar suatu tingkat kesulitan dapat dilihat dari nama dan [star rating](/wiki/Beatmapping/Star_rating) dari tingkat kesulitan tersebut.
+Sebuah **tingkat kesulitan beatmap** adalah sebuah level permainan yang tersimpan dalam bentuk teks di sebuah file dengan [ekstensi `.osu`](/wiki/osu!_File_Formats/Osu_(file_format)). Masing-masing level berisikan [hit object](/wiki/Hit_object) berdasarkan [mode permainan](/wiki/Game_mode) terkait yang diatur sesuai dengan [waktu (*timing*)](/wiki/Beatmapping/Timing_section) yang telah ditentukan. Masing-masing tingkat kesulitan memiliki aspek yang berbeda, yang secara langsung dapat mempengaruhi pengalaman pemain. Nama dari sebuah tingkat kesulitan dan [*star rating*](/wiki/Beatmapping/Star_rating) biasanya memberikan cukup informasi bagi pemain, apakah ini menantang atau tidak.
```

Comment:
```suggestion
Sebuah **tingkat kesulitan beatmap** adalah sebuah level permainan yang tersimpan dalam bentuk teks di sebuah file dengan [ekstensi `.osu`](/wiki/osu!_File_Formats/Osu_(file_format)). Masing-masing level berisikan [hit object](/wiki/Hit_object) berdasarkan [mode permainan](/wiki/Game_mode) terkait yang diatur sesuai dengan [waktu (*timing*)](/wiki/Beatmapping/Timing_section) yang telah ditentukan. Masing-masing tingkat kesulitan memiliki aspek yang berbeda, yang secara langsung dapat memengaruhi pengalaman pemain. Nama dari sebuah tingkat kesulitan dan [*star rating*](/wiki/Beatmapping/Star_rating) biasanya memberikan cukup informasi bagi pemain, apakah ini menantang atau tidak.
```
should be `memengaruhi` instead of `mempengaruhi`. https://kbbi.kemdikbud.go.id/entri/memengaruhi